packages/slate/test/interfaces/Operation/inverse/moveNode/child-to-parent.js

```diff
@@ -0,0 +1,9 @@
+import { Operation } from 'slate'
+
+export const input = { type: 'move_node', path: [0, 2, 1], newPath: [0, 2] }
+
+export const test = value => {
+  return Operation.inverse(value)
+}
+
+export const output = { type: 'move_node', path: [0, 2], newPath: [0, 3, 1] }
```

Comment:
should the path mutate here? `newPath: [0, 3, 1] `

Reply:
Yup, when the node is moved to `[0, 2]` it shifts what was its parent `[0, 2]` back inside its grandparent to `[0,3]`. I find it helpful to grab a whiteboard or piece of paper to draw these out when working on them.